benchmarks/CMakeLists.txt

```diff
@@ -32,7 +32,8 @@ function(ConfigureBench BENCH_NAME)
     ${BENCH_NAME}
     PROPERTIES POSITION_INDEPENDENT_CODE ON
                RUNTIME_OUTPUT_DIRECTORY "$<BUILD_INTERFACE:${RMM_BINARY_DIR}/gbenchmarks>"
-               CUDA_ARCHITECTURES "${CMAKE_CUDA_ARCHITECTURES}")
+               CUDA_ARCHITECTURES "${CMAKE_CUDA_ARCHITECTURES}"
```

Comment:
I don't see us installing benchmarks for other projects like cudf, so we shold drop this and stick with tests for now.